build/run

```diff
@@ -115,11 +115,11 @@ else
     fi
 fi
 
-# if DOWNLOADDIR is not set bind mount it
-if [[ -n "${DOWNLOADDIR}" ]]; then
-    mkdir -p ${DOWNLOADDIR}
-    DOWNLOAD_ARGS="-e DOWNLOADDIR=${BUILDER_HOME}/go/src/${source_repo}/.download \
-        -v ${DOWNLOADDIR}:${BUILDER_HOME}/go/src/${source_repo}/.download"
+# if CACHE_DIR is not set bind mount it
+if [[ -n "${CACHE_DIR}" ]]; then
+    mkdir -p ${CACHE_DIR}
+    DOWNLOAD_ARGS="-e CACHE_DIR=${BUILDER_HOME}/go/src/${source_repo}/.download \
```

Comment:
but we are still using .download here.  is that correct, or should it be .cache?